code/mob/living/carbon/human.dm

```diff
@@ -1397,7 +1397,7 @@
 
 	return rendered
 
-/mob/living/carbon/human/say(var/message, var/ignore_stamina_winded = 0)
+/mob/living/carbon/human/say(var/message, var/ignore_stamina_winded = 0, unique_maptext_style, maptext_animation_colors)
```

Comment:
```suggestion
/mob/living/carbon/human/say(var/message, var/ignore_stamina_winded = FALSE, var/unique_maptext_style = null, var/maptext_animation_colors = null)
```
As an extra thing, could you use `FALSE` instead of 0 for `ignore_stamina_winded`?